OJP/OJP_Trips.xsd

```diff
@@ -119,9 +119,14 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="BaseTripMobilityFilterGroup"/>
-			<xs:element name="LevelEntrance" type="xs:boolean" default="false" minOccurs="0">
+			<xs:element name="LevelEntranceOrBoardingAid" type="xs:boolean" default="false" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>The user needs vehicles with level entrance between  platform and vehicle, f.e. for wheelchair access.</xs:documentation>
+					<xs:documentation>The user needs vehicles with level entrance between platform and vehicle, an appropriate ramp, or assistance for boarding or unboarding (for assisted and unassisted wheelchairs, or similar constraints).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessibilityPreference" type="AccessibilityPreferenceEnumeration" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>If bestIfMobilityImpaired is chosen as OptimisationMethod, the algorithm tries to minimise the obstacles by also considering substantially longer Trips. The user may specify access features to be avoided. The routing algorithm tries to minimise the occurrences of the listed access features, the order of priority is given by the sequence in which they are listed.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
```

Comment:
```suggestion
			<xs:element name="AccessibilityPreferences" minOccurs="0">
				<xs:annotation>
					<xs:documentation>If bestIfMobilityImpaired is chosen as OptimisationMethod, the algorithm tries to minimise the obstacles by also considering substantially longer Trips. The user may specify access features to be avoided. The routing algorithm tries to minimise the occurrences of the listed access features, the order of priority is given by the sequence in which they are listed.</xs:documentation>
				</xs:annotation>
				<xsd:list itemType="AccessibilityPreferenceEnumeration" />
			</xs:element>
```

Reply:
@sgrossberndt @ue71603  no food before comments!

Reply:
ok for me.